Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -2318,7 +2364,14 @@ private static bool HandleKeywords(Material destMat, string keyword)
         Matrix4x4 localMatrix = objTransform.localToWorldMatrix;
         for (int x = 0; x < 3; x++)
         for (int y = 0; y < 3; y++)
+        {
+            if (Math.Abs(localMatrix[x, y] - converted[x, y]) > 0.01f)
```

Comment:
What is this for ?
Can this be separated into a different PR ?

Reply:
Deleted now